src/compileTranslations.test.ts

```diff
@@ -86,11 +88,23 @@ it('grabs .po files from any directory', (done) => {
     .pipe(expectTranslationsForLanguages(done, 'en-US', 'es-ES', 'fr-FR'));
 });
 
-function compileExpectError(defaultLanguage: string, done: jest.DoneCallback) {
+function compileExpectError(
+  defaultLanguage: string,
+  done: jest.DoneCallback,
+  // tslint:disable-next-line: max-union-size
+  expectedError?: Pick<PluginError, 'fileName' | 'message' | 'name' | 'plugin'>,
```

Comment:
This handy function forced to use snapshot testing. But one of the snapshot contains a runtime generated path that will change if you run this test on different platforms.
So I added an optional argument to pass here directly the `expectedError` and avoid to check by snapshot